### PR TITLE
Tbh 492 cci project build

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: tbconnect/tasks.py [hunk 1]
```diff
@@ -153,10 +153,12 @@ def send_tbcheck_data_to_cci(data):
         if (
             response.status_code == 200
             and b'"Received Successfully"' == response.content
+            or response.status_code == 200
+            and b'"Received Sucessfully"' == response.content
         ):
             return "CCI data submitted successfully"
         response.raise_for_status()
-        return "CCI data Submission failed"
+        return "CCI data Submission failed {}".format(response.content)
```
Comment (erikh360) [inline tbconnect/tasks.py]: Looking at this now I think this should also raise an exception so that we see it in sentry

File: tbconnect/tasks.py [hunk 2]
```diff
@@ -153,10 +153,12 @@ def send_tbcheck_data_to_cci(data):
         if (
             response.status_code == 200
             and b'"Received Successfully"' == response.content
+            or response.status_code == 200
+            and b'"Received Sucessfully"' == response.content
```
Comment (erikh360) [inline tbconnect/tasks.py]: Is it sometimes spelled correctly and sometimes not?

**Review by erikh360 — COMMENTED** [review]
Looks good but 1 comment and 1 question

File: tbconnect/tasks.py (hunk 2 above)
Comment (Buhle79) [reply tbconnect/tasks.py]: I once received the correct spelled but now i'm getting the missed spelled on.

Comment (Buhle79) [reply tbconnect/tasks.py]: I will stick to the latest one that i received

**Review by erikh360 — APPROVED** [review]

**Pull request merged**